pkg/virt-handler/non-root.go

```diff
@@ -128,12 +128,12 @@ func getTapDevices(vmi *v1.VirtualMachineInstance) ([]string, error) {
 }
 
 func (d *VirtualMachineController) prepareTap(vmi *v1.VirtualMachineInstance, res isolation.IsolationResult) error {
-	tapDevices, err := getTapDevices(vmi)
+	networkToTapDeviceNamesMap, err := getTapDevices(vmi)
 	if err != nil {
 		return err
 	}
-	for _, tap := range tapDevices {
-		path, err := isolation.SafeJoin(res, "sys", "class", "net", tap, "ifindex")
+	for networkName, tap := range networkToTapDeviceNamesMap {
```

Comment:
If one is called `networkName`, then you better be consistent and call the second `tapName` as well.